code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -66,10 +72,11 @@
 				break
 
 			if(Target in view(1,src))
+				var/feedcooldown = (transformeffects & SLIME_EFFECT_SEPIA) ? 2.5 SECONDS : 4.5 SECONDS // TEGU
 				if(!CanFeedon(Target)) //If they're not able to be fed upon, ignore them.
 					if(!Atkcool)
 						Atkcool = TRUE
-						addtimer(VARSET_CALLBACK(src, Atkcool, FALSE), 4.5 SECONDS)
```

Comment:
Instead of deleting /TG/ lines of code, it would be better to just comment out the entire block of code, and have a Tegu version of it, under.

Reply:
It doesn't make much sense.

Reply:
Still, doesn't make any sense when we're replacing it, instead of simply removing/commenting out.